src/json-patch-duplex.js

```diff
@@ -612,3 +609,6 @@ if (typeof exports !== "undefined") {
     exports.validator = jsonpatch.validator;
     exports.JsonPatchError = jsonpatch.JsonPatchError;
 }
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
hmmm, I don't know, changing it would break ES6 import. It produces this:
![image](https://cloud.githubusercontent.com/assets/17054134/21401736/aeef7dd8-c7b5-11e6-871f-012d2aa48592.png)

And here is the module code:
![image](https://cloud.githubusercontent.com/assets/17054134/21401756/c3ff098c-c7b5-11e6-9e53-f60ce75a6046.png)

Making it `exports.default = jsonpatch` will produce the first error too.